### PR TITLE
Feature slice

### PR DESCRIPTION
**Pull request by Meowmir**
## Describe your changes
Created a function which takes an array and returns a sliced array based on parameter start (int) and end (int).

## Code Usage examples
```js
const numbers = [1, 2, 3, 4]
numbers.slice(1, 3)
// console: [ 2, 3 ]
numbers.slice(-1)
// console: [ 4 ]
```

## Checklist before requesting a review
- [x] I have added JsDoc comments to all public methods created
- [x] EsLint is passing for this PR
- [x] Tests are passing for this PR
- [x] CI build is passing for this PR
- [x] I have performed a self-review of my code
- [x] I have had another developer review my code

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Mr-Squirrel) [inline .vscode/settings.json]: please remove this file from git and add to .gitignore

File: slice.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/**
+ * Creates a subarray which starts at optional second argument and ends before the optional third argument.
+ * @param {array} array - The array to be sent into the function.
+ * @param {int} start - The index where to start the slice.
+ * @param {int} end - The index where to end the slice.
+ * @returns
+ */
+
+export default function slice(array, start, end) {
```
Comment (Mr-Squirrel) [inline slice.js]: Please stop naming your parameters/variables after their data type. Use more semantic naming such as `toSlice` or `targetArr`

**Review by Mr-Squirrel — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (Meowmir) [reply .vscode/settings.json]: done

File: slice.js (hunk 2 above)
Comment (Meowmir) [reply slice.js]: changed parameter name to `toSlice`

**Comment by Meowmir** [comment]
@Mr-Squirrel Thank you for your review. I've updated the PR. Please review again.

**Review by Mr-Squirrel — APPROVED** [review]

**Pull request merged**